.circleci/config.yml

```
@@ -52,10 +52,14 @@ jobs:
             aws s3 cp s3://com-stitchdata-dev-deployment-assets/environments/tap-tester/tap_tester_sandbox tap-tester.env
             source tap-tester.env
             wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | apt-key add -
-            echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-4.2.list
+            echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" | \
+                 tee /etc/apt/sources.list.d/mongodb-org-4.2.list
```

For breaking up pipelines, I like to have the pipe be the first character on the subsequent lines. This makes it clear this is a pipeline instead of accidental whitespace before the second line
```suggestion
            echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/4.2 multiverse" \
                | tee /etc/apt/sources.list.d/mongodb-org-4.2.list
```

---

More clearly

```shell
echo "line one" | \
    echo "line two could look like random whitespace"
```

```shell
echo "line one" \
    | echo "line two"
```